src/main/webapp/app/pages/evaluation/application-overview/application-overview.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="body">
+  <jhi-dynamic-table
+    [columns]="columns"
+    [data]="pageData()"
+    [totalRecords]="total()"
+    [rows]="pageSize()"
+    [selectAble]="true"
```

Comment:
same comment here for "selectable"
```suggestion
    [selectable]="true"
```